python_modules/dagster/dagster/core/storage/tags.py

```diff
@@ -42,8 +42,9 @@
 DOCKER_IMAGE_TAG = "{prefix}image".format(prefix=SYSTEM_TAG_PREFIX)
 
 MAX_RETRIES_TAG = "{prefix}max_retries".format(prefix=SYSTEM_TAG_PREFIX)
+REEXECUTION_STRATEGY_TAG = "{prefix}reexecution_strategy".format(prefix=SYSTEM_TAG_PREFIX)
```

Comment:
was retry_strategy or job_retry_strategy on the table? reexecution is kind of a mouthful